.storybook/preview.tsx

```diff
@@ -85,8 +86,8 @@ export const globalTypes = {
     toolbar: {
       icon: 'graphline',
       items: [
-        { value: 'on', left: 'Analytics', title: 'Log to Actions' },
-        { value: 'off', left: 'Analytics', title: 'Off' },
+        { value: 'on', title: 'Log to Actions' },
+        { value: 'off', title: 'Off' },
```

Comment:
Maybe we should just set this to `on` by default?

Reply:
Yeah, I guess it wouldn't hurt. Would be useful to see that feedback by default